tests/rt.conf

```diff
@@ -1,325 +1,395 @@
-
-RUN     | cpld_bmark_p8                                                                                                           | - s4.intel jet.intel cheyenne.intel acorn.intel  | fv3 |
-RUN     | cpld_restart_bmark_p8                                                                                                   | - s4.intel jet.intel cheyenne.intel acorn.intel  |     | cpld_bmark_p8
-
-COMPILE | -DAPP=S2SW -DCCPP_SUITES=FV3_GFS_v17_coupled_p8                                                                         |                                         | fv3 |
-RUN     | cpld_control_noaero_p8                                                                                                  |                                         | fv3 |
-RUN     | cpld_control_nowave_noaero_p8                                                                                           |                                         | fv3 |
-
-COMPILE | -DAPP=S2SWA -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8,FV3_GFS_cpld_rasmgshocnsstnoahmp_ugwp                       | - wcoss2.intel acorn.intel              | fv3 |
-RUN     | cpld_debug_p8                                                                                                           | - wcoss2.intel acorn.intel              | fv3 |
-
-COMPILE | -DAPP=S2SW -DDEBUG=ON -DCCPP_SUITES=FV3_GFS_v17_coupled_p8                                                              | - wcoss2.intel acorn.intel              | fv3 |
-RUN     | cpld_debug_noaero_p8                                                                                                    | - wcoss2.intel acorn.intel              | fv3 |
+### RT.CONF FORMATTING ###
+# COMPILE Line ( Items separated by a | )
+# Item 1: COMPILE - This tells rt.conf the following information is to be used in setting up a compile job
+# Item 2: Compile number - must be sequential in rt.conf - use as a reference for compile failures
+# Item 3: Compiler to use in build (intel or gnu)
+# Item 4: CMAKE Options - Provide all CMAKE options for the build
+# Item 5: Machines to run on (- is used to ignore specified machines, + is used to only run on specified machines) 
+## -> EX: + hera orion gaea = compile will only run on hera orion and gaea machines 
+## -> EX: - wcoss2 acorn = compile will NOT be run on wcoss2 or acorn
+# Item 6:
+#
+# RUN Line ( Items separated by a | )
+## NOTE: The build resulting from the COMPILE line above the RUN line will be used to run the test 
+# Item 1: RUN - This tells rt.conf the following information is to be used in setting up a model run
+# Item 2: Test name - Which test in the tests/tests directory should be sourced?
+# Item 3: Machines to run on (- is used to ignore specified machines, + is used to only run on specified machines) 
+## reference example above
+# Item 4: Controls whether the run creates its own baseline or it uses the baseline from a different (control) test
```

Comment:
Add Item 5 which specifies the name of the test this test depends on.

Reply:
How do you describe the 5th column? When the 4th column is blank, then it compares against a control baseline, right? The 5th column is really where the test "depends" on another test.

Reply:
Yes. The 5th column is the name of the (previous) test that must be completed before this one starts, that's all . It is typically used for restart tests which depend on the output of the previous (cold start) test to be available. Whether the 4th column is blank or not does not matter, for restart cases this is mostly true, because restart test compare against it's 'dependent' test baselines, but conceivably one can decide to have some baseline files saved even for restart cases. So, basically this column is only used to specify the execution order of tests nothing else.

Reply:
Addressed in recent commit